FindOnPage.md

```diff
@@ -0,0 +1,619 @@
+        public CoreWebView2FindDirection FindDirection { get; set; }
+        public bool IsCaseSensitive { get; set; }
+        public bool ShouldMatchWord { get; set; }
+    }
+}
+```
+
+### CoreWebView2 Find Interface
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
+{
+    /// Specifies the direction of Search Parameters.
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    enum CoreWebView2FindDirection
+    {
+        /// Specifies a forward search in the document.
+        Forward, 
+        /// Specifies a backwards search in the document.
+        Backward, 
+    };
+
+    runtimeclass CoreWebView2FindConfiguration : [default]ICoreWebView2FindConfiguration {}
+
+    /// Interface defining the find configuration.
+    /// This interface provides the necessary methods and properties to configure a search operation.
+    [com_interface("staging=ICoreWebView2StagingFindConfiguration")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2FindConfiguration 
+    {
+        // Gets or sets the search term used for the find operation. Returns the search term.
+        String FindTerm { get; set; };
+        // Gets or sets the direction of the search operation (forward or backward). Returns the search direction.
+        CoreWebView2FindDirection FindDirection { get; set; };
+        // Determines if the search operation is case sensitive. Returns TRUE if the search is case sensitive, FALSE otherwise.
+        Boolean IsCaseSensitive { get; set; };
+        // Determines if only whole words should be matched during the find operation. Returns TRUE if only whole words should be matched, FALSE otherwise.
+        Boolean ShouldMatchWord { get; set; };
+    };
+
+    runtimeclass CoreWebView2Find : [default]ICoreWebView2Find {}
+
+    /// Interface providing methods and properties for finding and navigating through text in the web view.
+    /// This interface allows for text searches, navigation between matches, and customization of the find UI.
+    [com_interface("staging=ICoreWebView2StagingFind")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2Find 
+    {
+        [completed_handler("")]
+        /// Initiates a search using the specified configuration.
+        /// Displays the Find bar and starts the search operation. If a search was already ongoing, it will be stopped and replaced with this new instance.
+        /// If called with an empty string, the Find bar is displayed but no search occurs. Changing the configuration object after initiation won't affect the ongoing search.
+        /// To change the ongoing search, StartFind must be called again with a new or modified configuration object.
+        /// This method is primarily designed for HTML document searches.
+        Windows.Foundation.IAsyncAction StartFindAsync(CoreWebView2FindConfiguration configuration);
```

Comment:
The async action completes for start find when the UI has been displayed with the find term in the UI bar, and the matches have populated on the counter on the find bar. When the UI is displayed, there can be a little bit of latency between the searches populating in the UI (the number assocaisted with match index and match count changing from 0/0 to their respective numbers), but In terms of matchcountchanged and activematchindex, those events are ONLY raised after startfindasync has completed, otherwise they will have their default values (-1 for both).